### PR TITLE
Feature/e UI 8669

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8669

### Change description ###

Register Org

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by johnbenjamin-hmcts — DISMISSED** [review]

File: src/register-org/services/register-org.service.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { SessionStorageService } from '../../../src/shared/services/session-storage.service';
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/register-org/services/register-org.service.ts]: Removed

File: src/register-org/register-org.module.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { SharedModule } from '../shared/shared.module';
+import { registerRouting } from './register-org.routing';
+import * as fromServices from './services';
+import { HttpClientModule } from '@angular/common/http';
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/register-org/register-org.module.ts]: Fixed the ordering

File: src/register-org/containers/register/register-org.component.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { RegisterOrgService } from '../../services/index';
+import { RegistrationData } from '../../models/registrationdata.model';
```
[the remark this answers is not in the document]
Comment (udaydenduluri33) [reply src/register-org/containers/register/register-org.component.ts:3]: Fixed

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Pull request merged**